### ISSUE TITLE
Define how properties with tuples can be interpreted

### ISSUE THREAD

**Issue by jonaslagoni** [json-schema-org/vocab-idl#21]
We need to define how properties with tuples should be interpreted, such as for the following schema:

{
  "$schema": "https://json-schema.org/draft/2020-12/idl-schema",
  "name": "SomeTitle",
  "type": "object",
  "properties": {
    "ArrayProperty": {
      "type": "array",
       "prefixItems": { "type": "string" },
       "items": false
    }
  }
}

**Comment by jdesrosiers** [comment]
> should we expect an external class to be created?

Yes. It could be either a one-off class specifically for that type or a generic tuple class(s) that can be reused for any tuples. I'd prefer the later.
